_includes/sign_in.html

```diff
@@ -2,7 +2,7 @@
   class="{{include.mobile}} usa-button sign-in-logo"
   href="{{ site.idp_base_url }}/{% if page.lang!= 'en' %}{{page.lang}}{% endif %}"
   title="Login.gov"
-  aria-label="Login.gov"
+  aria-label="{{ site.data.[page.lang].settings.nav.sign_in }} Login.gov"
```

Comment:
Do we need the `aria-label` at all, or can it be removed? Based on the text, I would assume it would have the same computed value.

```suggestion
```

Reply:
I had initially worried that the title='Login.gov' would interfere, but without the aria-label, mac voiceover it's reading it as `link; Sign in with Login.gov, Login.gov`. That seems good to me.
Should I be testing with other accessibility tools in addition to voiceover?

Reply:
Ah, I hadn't seen the `title` attribute. I think that could be okay, but I might go a step further and suggest we remove the `title` attribute as well, since I don't think it's really adding anything for us, and its use can be quite [problematic](https://www.a11yproject.com/posts/title-attributes/).

Reply:
Good point. I removed the title here as well as two other redundant titles I found in a quick search.

Reply:
>Should I be testing with other accessibility tools in addition to voiceover?

Sorry I skipped over this question. There's a good article in the TTS Accessibility guide that covers testing tools:

https://accessibility.18f.gov/tools/

Generally, VoiceOver is a good start. The Accessibility Insights and/or Deque browser extensions are quite good as well. We also have some automated tests as part of the CI build via [end-to-end tests](https://github.com/18F/identity-site/blob/main/spec/e2e/accessibility_spec.js).